tidb-in-kubernetes/maintain/restart.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Restart TiDB Cluster in Kubernetes
+summary: Learn how to restart the TiDB cluster in the Kubernetes cluster.
+category: how-to
+---
+
+# Restart TiDB Cluster in Kubernetes
+
+This document describes how to forcibly restart the TiDB cluster in the Kubernetes cluster, including restarting a Pod, restarting all Pods of a component, and restarting all Pods of the TiDB cluster.
+
+> **Note:**
+>
+> TiDB Operator v1.0.x only supports restarting the Pod in a forcible way.
+>
+> - If the PD Pod being restarted is the PD leader, its leadership does not automatically shift, causing the PD service to be momentarily unavailable.
```

Comment:
- Perhaps "temporarily" is more commonly used in our doc.
- In this note, we need to put emphasis on the result, so I suggest the following changes.

```suggestion
> - If the PD Pod being restarted is the PD leader, its leadership does not automatically shift, which can cause the PD service to be temporarily unavailable.
```